external/netfx/netfx.depproj

```diff
@@ -3,8 +3,17 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
-    <NuGetTargetMoniker Condition="'$(TargetGroup)' == 'netcoreapp'">.NETFramework,Version=v4.6.1</NuGetTargetMoniker>
     <NugetRuntimeIdentifier>None</NugetRuntimeIdentifier>
+    <OutputPath>$(RefRootPath)/$(TargetGroup)</OutputPath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetGroup)' == 'net463'">
+    <!-- Missing 4.6.3 targeting pack, use 462 for now -->
+    <TargetingPackNugetPackageId>Microsoft.TargetingPack.NETFramework.v4.6.2</TargetingPackNugetPackageId>
```

Comment:
I see you are now making use of TargetingPackNugetPackageId I was planning to clean that out, but I guess I will leave it alone now.